src/components/MainPage.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable */
+import restaurants from '../data/restaurants.json'
+
+function MainPage() {
```

Comment:
лучше делай через const, а не через function. Так принято в большинстве проектов

```suggestion
const MainPage = () => {
```